src/blocks/FooterBlock/MarketPicker/MarketPicker.tsx

```diff
@@ -1,70 +1,133 @@
 import styled from '@emotion/styled'
-import React from 'react'
-import { MOBILE_BP_UP } from 'components/blockHelpers'
-import { Select } from 'components/Select/Select'
+import React, { useRef, useState, useCallback } from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
 import {
   getMarketsInLocalLang,
   getMarketsInEnglish,
   checkIsInEnglish,
   locales,
   LocaleData,
 } from 'utils/locales'
-import { MarketOption } from './MarketOption'
-import { MarketValueContainer } from './MarketValueContainer'
+import { ButtonBrandPivot } from 'components/ButtonBrandPivot/Button'
+import { Globe } from 'components/icons/Globe'
+import { Chevron } from 'components/icons/Chevron'
+import { MOBILE_BP_UP } from 'components/blockHelpers'
+import { useClickOutside } from './useClickOutside'
+
+const Wrapper = styled.div`
+  position: relative;
+`
+
+const Button = styled(ButtonBrandPivot)`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  max-width: 15rem;
 
-const MarketSelect = styled(Select)`
   ${MOBILE_BP_UP} {
-    max-width: 15rem;
+    width: 100%;
+  }
+`
+
+const GlobeIcon = styled(Globe)`
+  width: 1.5rem;
+  height: 1.5rem;
+  flex-shrink: 0;
+  margin-left: -0.875rem;
+`
+
+const ButtonLabel = styled.div`
+  flex: 1;
+  padding-left: 0.5rem;
+  text-align: left;
+`
+
+const Menu = styled.ul`
+  position: absolute;
+  width: 100%;
+  background: ${colorsV3.gray900};
+  border: 1px solid ${colorsV3.white};
+  border-radius: 8px;
+  padding: 0;
+  margin: 0.5rem 2px 0;
+  max-width: 15rem;
+`
+
+const MenuItem = styled.li`
+  list-style: none;
+  height: 3rem;
+  padding: 0.75rem 1.5rem;
+`
+
+const MenuItemLink = styled.a`
+  text-decoration: none;
+  display: block;
+  height: 100%;
+
```

Comment:
Maybe put the padding on the `a` instead of on the `li` to make the entire area clickable? 
```suggestion
const MenuItem = styled.li`
  list-style: none;
  height: 3rem;
`

const MenuItemLink = styled.a`
  text-decoration: none;
  display: block;
  height: 100%;
  padding: 0.75rem 1.5rem;

```

Reply:
Also, should this really be an `a` and not a `Link` from React Router? Or is it because we want to add the `hrefLang` attribute?